fsw/src/mm_dump.c

```diff
@@ -517,6 +518,7 @@ bool MM_DumpInEventCmd(const CFE_SB_Buffer_t *BufPtr)
                     ** 13 characters, not counting NUL terminator
                     */
                     strncpy(EventString, "Memory Dump: ", CFE_MISSION_EVS_MAX_MESSAGE_LENGTH);
```

Comment:
"Memory Dump: " may exceed length of `CFE_MISSION_EVS_MAX_MESSAGE_LENGTH` when `CFE_MISSION_EVS_MAX_MESSAGE_LENGTH` is configured less than size 13.